### PR TITLE
Chan agent to app agent pool

### PR DESCRIPTION
**Pull request by Pazkov**
I have changed something on events and properties of Agents to be compliant with new module app_agent_pool in substitution of removed chan_agent module.

### CODE REVIEW

File: Asterisk.2013/Asterisk.NET/Manager/Event/AgentConnectEvent.cs [hunk 1]
```diff
@@ -17,6 +17,7 @@ public AgentConnectEvent(ManagerConnection source)
 
         /// <summary>
         ///     Get/Set bridged channel.
```
Comment (Deantwo) [inline Asterisk.2013/Asterisk.NET/Manager/Event/AgentConnectEvent.cs]: Missing `<br />`.

File: Asterisk.2013/Asterisk.NET/Manager/Action/QueueAddAction.cs [hunk 2]
```diff
@@ -94,5 +94,11 @@ public override string Action
         ///     true if the queue member should be paused when added.
         /// </summary>
         public bool Paused { get; set; }
+
+        /// <summary>
+        ///     Get/Set an alternate interface to be used to determine the state of the member.
```
Comment (Deantwo) [inline Asterisk.2013/Asterisk.NET/Manager/Action/QueueAddAction.cs]: Missing `<br />`.

**Comment by Pazkov** [comment]
Hi Deantwo,
I’ve updated the comments.
We are testing this changes since two month in an internal production environment and no issues was reported.

**Comment by Deantwo** [comment]
Only thing I aren't sure about is if any of the inheritance is done correct, but I don't see any mistakes on my skim.

To see what I mean you can check the issue we had with Queue events.
Here: https://github.com/AsterNET/AsterNET/issues/128

**Comment by Pazkov** [comment]
Hi Deantwo,
any news from skrusty? I'd like to test the new Async features, but this changes are not yet merged...

**Comment by skrusty** [comment]
Looks fine to me

**Comment by Pazkov** [comment]
Sorry,
I know that Deantwo merged moldypenguins:async/await 24 days ago but I can't use this without my pull request.
I'm talking about to merge my pull request in master too. 
Deantwo is waiting for skrusty review of my work, I think.

**Pull request merged**